lib/processor.js

```diff
@@ -12,6 +12,9 @@ var assign = require("object-assign");
 var remark = require("remark");
 
 var SUPPORTED_SYNTAXES = ["js", "javascript", "node", "jsx"];
+var UNSATISFIABLE_RULES = [
```

Comment:
Hmm... You could use `settings`. Although I'm having trouble finding it in the documentation for some reason.
Ah, found it (yay, search!), it's strange that it's not mentioned in working with plugins, but it is mentioned in the configuring eslint: http://eslint.org/docs/user-guide/configuring#adding-shared-settings